ttyplot.c

```diff
@@ -183,7 +187,7 @@ int main(int argc, char *argv[]) {
                 break;
         }
 
-    initscr(); /* uses filesystem, so before pledge */
+    sp = newterm("linux", stdout, stdin);
```

Comment:
This definitely should not be "linux" type. You are hardcoding terminal type. Null perhaps?

Reply:
I'm also now thinking that with hardcoded terminal type your test results are probably not conclusive.